cmd/sdkch/README.md

```diff
@@ -0,0 +1,174 @@
+# sdkch
+Simple tool to maintain modular changelogs
+
+# Usage
+
+```
+$ sdkch -help
+usage: sdkch COMMAND
+
+Maintain unreleased changelog entries in a modular fashion.
+
+Commands:
+    init
+    add SECTION STANZA            Add an entry file.
+                                  Read from stdin until it
+                                  encounters EOF.
+    generate [VERSION]            Generate a changelog in
+                                  Markdown format and print it
+                                  to stdout. VERSION defaults
+                                  to UNRELEASED.
+
+    Sections             Stanzas
+         ---                 ---
+    breaking                gaia
+    features             gaiacli
+improvements            gaiarest
+    bugfixes                 sdk
+                      tendermint
+
+Flags:
+  -d string
+    	entry files directory (default "./.pending")
+```
+
+## Initialise a new entry files directory
+
+First, you need to initialise a new directory that will host entry files:
+
+```bash
+$ sdkch init
+```
+
+A new hidden directory `.pending` is created. Its 2-level structure looks like the following:
+
+```
+$ tree .pending/
+.pending/
+|-- breaking
+|   |-- gaia
+|   |-- gaiacli
+|   |-- gaiarest
+|   |-- sdk
+|   `-- tendermint
+|-- bugfixes
+|   |-- gaia
+|   |-- gaiacli
+|   |-- gaiarest
+|   |-- sdk
+|   `-- tendermint
+|-- features
+|   |-- gaia
+|   |-- gaiacli
+|   |-- gaiarest
+|   |-- sdk
+|   `-- tendermint
+`-- improvements
+    |-- gaia
+    |-- gaiacli
+    |-- gaiarest
+    |-- sdk
+    `-- tendermint
+
+24 directories, 0 files
+```
+
+Deepest level directories can contain `entry files`. Entry files are created before submitting a PR for review.
+
+## Add a new entry
+
+You can either drop a text file in the appropriate directory or use the `add` command:
+
+```bash
+$ sdkch add features sdk
```

Comment:
This command is tremendously frustrating to use. Once you type in `sdkch add features sdk` if just waits for an endless number of new line inputs with no clear way to exit the process. Not to mention, if you mess up a line, you cannot go back to the previous line (horrid). I'd strongly recommend that an editor such as `vim` is opened _at_ the new file such that entries can be manually added in ... that or we simply do not allow for multiline input in this manner (which seems to be broken). 

Anyways yeah, this add example doesn't work

Reply:
Done